pkg/resources/stream_test.go

```diff
@@ -1,238 +1,239 @@
 package resources_test
 
 import (
+	"context"
 	"database/sql"
+	"database/sql/driver"
 	"testing"
 	"time"
 
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+
 	sqlmock "github.com/DATA-DOG/go-sqlmock"
 	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/provider"
 	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/resources"
-	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/snowflake"
 	. "github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/testhelpers"
 	"github.com/stretchr/testify/require"
 )
 
+var streamColumns = []string{"name", "database_name", "schema_name", "owner", "comment", "table_name", "type", "stale", "mode", "source_type"}
+
+func streamRow(mode string) []driver.Value {
+	return []driver.Value{"stream_name", "database_name", "schema_name", "owner_name", "grand comment", "target_table", "DELTA", false, mode, "Table"}
+}
+
 func TestStream(t *testing.T) {
-	r := require.New(t)
 	err := resources.Stream().InternalValidate(provider.Provider().Schema, true)
-	r.NoError(err)
+	require.NoError(t, err)
 }
 
 func TestStreamCreate(t *testing.T) {
-	r := require.New(t)
-
-	in := map[string]interface{}{
+	d := stream(t, "database_name|schema_name|stream_name", map[string]any{
 		"name":              "stream_name",
 		"database":          "database_name",
 		"schema":            "schema_name",
 		"comment":           "great comment",
-		"on_table":          "target_db.target_schema.target_table",
+		"on_table":          "\"target_db\".\"target_schema\".\"target_table\"",
```

Comment:
would be good if we could support both ways. With and without double quotes. Some people don't like using double quotes.

Reply:
I just saw in the other pr and somewhere else that we're using the approach with \". I agree we should support both (but imo a way with \" is so bad). I'll adjust the decoder, so It'll parse both

Reply:
Yes, and this is a bit of breaking change if we stop to support these now, right?